### PR TITLE
feat(fabric): add `Gamma` & `Vibrance` filters #62371

### PR DESCRIPTION
**Pull request by oxwazz**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<[Gamma](http://fabricjs.com/docs/fabric.Image.filters.Gamma.html), [Vibrance](http://fabricjs.com/docs/fabric.Image.filters.Vibrance.html)>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/fabric/test/index.ts
1. types/fabric/fabric-impl.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9d26c395164be0cecddc9e07d4288aeabd15cb96
</p>

File: types/fabric/fabric-impl.d.ts [hunk 1]
```diff
@@ -5492,6 +5492,30 @@ interface IAllFilters {
          */
         fromObject(object: any): ITintFilter;
     };
+    Gamma: {
```
Comment (RyanCavanaugh) [inline types/fabric/fabric-impl.d.ts:5495]: This is the only option in the entire body that isn't in alphabetic sorted order - probably best to put it in its correct place

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

File: types/fabric/fabric-impl.d.ts (hunk 1 above)
Comment (oxwazz) [reply types/fabric/fabric-impl.d.ts]: thank your for the correction, i've push latest commit with sorted alphabetical order, please review

**Comment by andrewbranch** [comment]
> - [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <>

@oxwazz this field is required, especially for popular packages. I’d like to review this and get it merged, but I have nothing to reference 🥲

**Comment by oxwazz** [comment]
> > * [ ]  Provide a URL to documentation or source code which provides context for the suggested changes: <>
> 
> @oxwazz this field is required, especially for popular packages. I’d like to review this and get it merged, but I have nothing to reference 🥲

i'm sorry, i already added the documentation reference on that field. is this enough @andrewbranch ?

thank you 👍

**Review by andrewbranch — APPROVED** [review]
Thank you!

**Comment by oxwazz** [comment]
Ready to merge

**Pull request merged**